spec/helpers/locale_helper_spec.rb

```diff
@@ -33,7 +33,7 @@
 
     it "returns default locale for a string with no locale translation" do
       I18n.with_locale(:cy) do
-        expect(t_locale_fallback("formats.big_decision", count: 1, locale: :cy)).to eq(:en)
+        expect(t_locale_fallback("content_item.schema_name.case_study.no_locale", count: 1, locale: :cy)).to eq(:en)
```

Comment:
Is this right? I think we want the case_study locales to go into `formats.case_study.<whatever>`